experiment/measurer/coverage_utils.py

```diff
@@ -271,26 +271,29 @@ def generate_json_summary(coverage_binary,
     return result
 
 
-def extract_covered_regions_from_summary_json(summary_json_file):
-    """Returns the covered regions given a coverage summary json file."""
-    covered_regions = []
+def extract_covered_branches_from_summary_json(summary_json_file):
+    """Returns the covered branches given a coverage summary json file."""
+    covered_branches = []
     try:
         coverage_info = get_coverage_infomation(summary_json_file)
         functions_data = coverage_info['data'][0]['functions']
-        # The fourth number in the region-list indicates if the region
-        # is hit.
-        hit_index = 4
-        # The last number in the region-list indicates what type of the
-        # region it is; 'code_region' is used to obtain various code
-        # coverage statistic and is represented by number 0.
+
+        # The fourth and the fifth item tell whether the branch is evaluated to
```

Comment:
Accorded to [llvm-cov indexing](https://github.com/llvm/llvm-project/blob/main/llvm/tools/llvm-cov/CoverageExporterJson.cpp#L93): the fourth and the fifth items are chosen because we want to know if the branch is evaluated and also whether it is evaluated to True of False.